scss/_base_icon-definitions.scss

```diff
@@ -74,6 +74,10 @@
   background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='16' height='16'%3E%3Cg fill='none' fill-rule='nonzero'%3E%3Cpath fill='#{vf-url-friendly-color($bg-color)}' d='M8 1a7 7 0 110 14A7 7 0 018 1zm2.83 3.502L6.863 9.884 5.174 8.096l-1.09 1.03 2.92 3.096 5.034-6.83-1.208-.89z'/%3E%3Cpath fill='#{vf-url-friendly-color($fg-color)}' d='M10.83 4.502l1.208.89-5.033 6.83-2.922-3.096 1.091-1.03 1.689 1.789z'/%3E%3C/g%3E%3C/svg%3E");
 }
 
+@mixin vf-icon-dot($bg-color: $color-positive, $fg-color: $color-x-light) {
```

Comment:
This is not really "an icon", and we only use it in one place, so I'm not sure if it needs to be exposed as mixin now.

Even as mixin, I guess it should live in the list component for now, as that's the only place where it is used.

Reply:
it is a unicode symbol, not an icon - needs to come from the font, this way it will use the correct weight if we ever decide to change it, etc